src/main/java/step2/TextSeparator.java

```diff
@@ -0,0 +1,21 @@
+package step2;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class TextSeparator {
+    private static final String CUSTOM_SEPARATOR_REGEX = "//(.)\n(.*)";
+    private static final String DEFAULT_SEPARATOR_REGEX = ",|:";
+    public static final int CUSTOM_DELIMITER_GROUP = 1;
+    public static final int TEXT_GROUP = 2;
```

Comment:
```suggestion
    private static final int CUSTOM_DELIMITER_GROUP = 1;
    private static final int TEXT_GROUP = 2;
```

접근제한자를 변경해도 무관할 것 같습니다~